rero_ils/theme/views.py

```diff
@@ -151,6 +150,17 @@ def url_active(string, target):
     return string
 
 
+@blueprint.app_template_filter('viewOrganisationName')
+def view_organisation_name(viewcode):
+    """Get view name."""
+    if viewcode != current_app.config.get('RERO_ILS_SEARCH_GLOBAL_VIEW_CODE'):
+        org = Organisation.get_record_by_viewcode(viewcode)
+        if org:
+            return org['name']
+    else:
```

Comment:
We don't need the `else` here. In the case we have no  `org` this function will return `None`!